modules/generative-cohere/clients/cohere.go

```diff
@@ -73,14 +73,15 @@ func (v *cohere) Generate(ctx context.Context, cfg moduletools.ClassConfig, prom
 	if err != nil {
 		return nil, errors.Wrap(err, "join Cohere API host and path")
 	}
+	emptyChatHistory := make([]message, 0)
 	input := generateInput{
-		Prompt:            prompt,
-		Model:             settings.Model(),
-		MaxTokens:         settings.MaxTokens(),
-		Temperature:       settings.Temperature(),
-		K:                 settings.K(),
-		StopSequences:     settings.StopSequences(),
-		ReturnLikelihoods: settings.ReturnLikelihoods(),
+		ChatHistory:   emptyChatHistory,
```

Comment:
there's no need of passing an empty slice, just remove this and leave it empty it will work the same as `empty` slice assignment